src/msgwindow.h

```diff
@@ -38,7 +38,7 @@ enum MsgColors
 	COLOR_TEXT,		/**< Color for normal text.  Default is black. */
 	COLOR_MESSAGE,	/**< Color for ``geany-compiler-message``.  Default is blue. */
 
-#ifndef GEANY_DISABLE_DEPRECATED
+#ifndef GEANY_DISABLE_DEPRECATED /* 2021/11/24 */
```

Comment:
Thats yesterday in my timezone :wink:

actually good point but better expressed as "since 1.40" (or whatever the next version is) in each doxygen comment

Reply:
@elextr

> Thats yesterday in my timezone wink

I did open this PR yesterday, but it probably wouldn't be merged until next week.  I'll change the date to 2021/12.  (Having a date would show how much time has passed without having to look for at a release history.)

> better expressed as "since 1.40" (or whatever the next version is) in each doxygen comment

I'll add a version number.